src/drafts/MarkdownViewer/_useListInteraction.ts

```diff
@@ -4,6 +4,8 @@ import {ListItem, listItemToString, parseListItem} from '../MarkdownEditor/_useL
 
 type TaskListItem = ListItem & {taskBox: '[ ]' | '[x]'}
 
+const isCodeBlockDelimiter = (line: string) => line.trimStart().startsWith('```')
```

Comment:
The [spec](https://github.github.com/gfm/#fenced-code-blocks) is a little looser than this, so there will be some types of code blocks that don't get caught.

They can be delimited by tildes instead of backticks:

```md
~~~
example
~~~
```

They can also have more than three backticks:

`````md
````
example
````
`````

In this case, you'd need to keep track of how long the starting delimiter is so that you can ensure the ending delimiter is the same length.

Reply:
Great point! I think for this specific bug fix (preventing the buggy interaction between legacy task lists and tasklists), the backtick delimiter does the trick, but I'm open to making the fix more robust (checking for tildes) to cover more edge cases.